tasks/task450_emea_fr_sk_translation.json

```diff
@@ -0,0 +1,2764 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "emea fr-sk (dataset link- https://huggingface.co/datasets/emea)"
+    ],
+    "Categories": [
+        "Translation"
+    ],
+    "Input_language": "French",
+    "Output_language": "Slovak",
+    "instruction_language": "English",
```

Comment:
Are the above three fields needed? @swarooprm

Reply:
fine.

Reply:
ok